lib/default-theme/NavLinks.vue

```diff
@@ -1,11 +1,33 @@
 <template>
   <nav class="nav-links" v-if="userLinks.length || githubLink">
     <!-- user links -->
-    <router-link v-for="item in userLinks"
-      :to="item.link"
+    <div
+      class="nav-item"
+      v-for="item in userLinks"
       :key="item.link">
-      {{ item.text }}
-    </router-link>
+      <div
+        v-if="item.type === 'links'"
+        class="dropdown-wrapper">
+        <span class="dropdown-title">{{ item.text }}</span>
+        <span class="arrow"></span>
+        <ul class="nav-dropdown">
+          <li
+            v-for="subItem in item.items"
+            :key="subItem.link">
+            <h4 v-if="subItem.type === 'links'">{{ subItem.text }}</h4>
+            <ul v-if="subItem.type === 'links'">
+              <li
+                v-for="childSubItem in subItem.items"
+                :key="childSubItem.link">
+                <nav-link :item="childSubItem"></nav-link>
```

Comment:
Some conflicts due to fixes which are also addressed in this PR. Should be easy to resolve.

Reply:
All conflict has been resolved.